src/pages/settings/Profile/Contacts/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -136,12 +137,13 @@ function BaseValidateCodeForm(props) {
                 onClose={() => User.clearContactMethodErrors(props.contactMethod, 'validateCodeSent')}
             >
                 <View style={[styles.mt2, styles.dFlex, styles.flexColumn]}>
-                    <Text
-                        style={[styles.link, styles.mr1]}
+                    <TouchableOpacity
+                        disabled={props.network.isOffline}
                         onPress={resendValidateCode}
+                        style={[styles.mr1]}
```

Comment:
@parasharrajat @abdulrahuman5196 
I have two options here:
1. Update `styles.link` to be a function and recieve `isDisabled` argument and based on that return coresponding styles
2. Create new `styles.disabledLink` styles and apply it in each instance based on disabled state

Which one you think is better

Reply:
@rushatgabhane 
I think we shouldn't add offline indicator in contact methods for the case 2. 
For the 1st case above, I can modify usage of `TouchableOpacity` to apply `cursor: not-allowed` styles when disabled. But it should be applied for every instance. 

What do you think?

Reply:
@rushatgabhane . Can you respond to my last comment

Reply:
This line is responsible for this. 
https://github.com/Expensify/App/blob/3ce8e8212c8c9dcc4a7767a1c256555624ad6202/src/components/ScreenWrapper/index.js#L147

It is in `ScreenWrapper` component which is used almost everywhere, except signin pages

Reply:
@rushatgabhane 
One last thing. Applying `cursor: not-allowed` to link is not working since TouchableOpacity is applying `pointer-events: none` . Should I change `pointer-events` of `TouchableOpacity` in every instance. 
Here is the version without cursor changes

https://github.com/Expensify/App/assets/59907218/8446baf8-fe25-4234-b89c-162fb379a15c